lib/serialize.js

```diff
@@ -56,78 +58,79 @@ export async function pageProps({ params }) {
     data.title = ''
   }
 
-  const importBasePath = `${process.cwd()}/content${slug
-    .split('/')
-    .slice(0, -1)
-    .join('/')}`
+  const mdxPathAbsolute = stdpath.join(process.cwd(), 'content', route.path)
 
-  const mdxSource = await serialize(content, {
-    parseFrontmatter: false,
-    scope: { data },
-    format: process.env.DOCS_USE_MDX === 'true' ? 'mdx' : 'md',
```

Comment:
`format` is not a valid parameter of the next-mdx-remote/serialize.
 * https://github.com/hashicorp/next-mdx-remote#apis

It is a parameter of the MDX compile function, but it's not necessary if you tell the compiler which file extensions are mdx vs md, which is what I've done.

 * https://mdxjs.com/packages/mdx/#optionsformat